js/main.js

```diff
@@ -1,2 +1,179 @@
+
+  output += "<p>You played <u>"+playerChoice+"</u>,The bot played <u>"+botChoice+"</u>.<p/>"
+
+  if (playerChoice === botChoice)  {
+    output += "<p>You tied</p>"
+
+  }
+  // PLAYER LOSES
+  else if  ((playerChoice === "rock" && botChoice === "paper") || (playerChoice === "paper" && botChoice === "scissors") || (playerChoice === "scissors" && botChoice === "rock") ) {
+
+    output += "<p>You Lose :(</p>"
+    botScore = botScore + 1
+    // Bonus Requirement 3
+    botBonus = botBonus+ 1
+    playerBonus = 0
+
+      if (botBonus >= 3) {
+        botScore = botScore + 2
+        $('#bonus').html("<p>Bot got two bonus points for winning 3 hand in a row</p>")
+        botBonus = 0
+        playerBonus = 0
+    }
+
+  }
+
+  // PLAYER WINS
+
+  else if ((playerChoice === "paper" && botChoice === "rock") || (playerChoice === "scissors" && botChoice === "paper") || (playerChoice === "rock" && botChoice === "scissors") ) {
+
+    output += "<p>You Win :)</p>"
+    playerScore = playerScore + 1
+
+    // Bonus Requirement 3
+    playerBonus = playerBonus + 1
+    botBonus = 0
+
+    if (playerBonus >= 3) {
+
+      playerScore = playerScore + 2
+      $('#bonus').html("<p>You got two bonus points for winning 3 hands in a row</p>")
+      botBonus = 0
+      playerBonus = 0
+    }
+    // End Bonus Requirement 3
+  }
+
+// Display Message and Scores
+
+  $('#humanScore').text(playerScore)
+  $('#computerScore').text(botScore)
+  $('#status').html(output)
+
+
+// Bonus Requirement 1
+  if (playerScore >= 10 || botScore >= 10 ) {
+
+    $('#statusRound').text(" ")
+    $('.scoreboardround').show()
+    $('#statusround').show()
+    $('#playagain').show()
+    $(".choice-buttons").attr("disabled", true);
+
+    if (playerScore >= 10) {
+      playerRoundScore = playerRoundScore +1
+      $('#statusround').html("<p>You won the round with a score of: "+playerScore+"</p>")
+      $('#humanScoreRound').text(playerRoundScore)
+
+    }
+    if (botScore >= 10) {
+      botRoundScore = botRoundScore + 1
+      $('#statusround').html("<p>Bot won the round with a score of: "+botScore+"</p>")
+      $('#computerScoreRound').text(botRoundScore)
+    }
+  }
+// End Bonus Requirement 1
+
+// Bonus Requirement 2
+
+  if (playerRoundScore >= 2 || botRoundScore >= 2 ) {
+
+    $('#statusgame').text(" ")
+    $('.scoreboardgame').show()
+    $('#statusgame').show()
+    $(".choice-buttons").attr("disabled", true);
+
+    if (playerRoundScore >= 2) {
+      $('#statusgame').html("<p>You won the game by winning "+playerRoundScore+" rounds</p>")
+      $('#humanScoreGame').text(playerRoundScore)
+    }
+    if (botRoundScore >= 2) {
+      $('#statusgame').html("<p>Bot won the game by winning "+botRoundScore+" rounds</p>")
+      $('#computerScoreGame').text(botRoundScore)
+    }
+  $('#playagain').hide()
+  $('#reloadgame').show()
+}
+// End Bonus Requirement 2
+
+
+})
+
+$("#reset").on("click", function (e) {
+  event.preventDefault();
```

Comment:
@thetripler the `event` object is passed via the callback function, which in this case is `e`.
A great use case for handling bubbling